ledger/ledger-on-memory/BUILD.bazel

```diff
@@ -226,7 +226,7 @@ conformance_test(
         "--verbose",
         "--include=ParticipantPruningIT",
         # Disable tests targeting only append-only schema functionality
-        "--exclude=ParticipantPruningIT:PRLocalAndNonLocalRetroactiveDivulgences,ParticipantPruningIT:PRRetroactiveDivulgences",
+        "--exclude=ParticipantPruningIT:PRLocalAndNonLocalRetroactiveDivulgences,ParticipantPruningIT:PRRetroactiveDivulgences,ParticipantPruningIT:PRDisclosureAndRetroactiveDivulgence",
```

Comment:
Normalization of these conformance test will be done in a subsequent PR. For clarity/conciseness, in this PR, I blacklist the tests that cannot run on single-participant setups.